foreman_bootdisk.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
     'LICENSE',
     'README.md'
   ]
-  s.files = `git ls-files`.split("\n") - Dir['.*', 'Gem*', '*.gemspec']
+  s.files = `git ls-files`.split("\n") - Dir['.*', 'Gem*', '*.gemspec', 'package-lock.json']
```

Comment:
I think that because `package-lock.json` is in `.gitignore` there's no need to place it here as well because git ls-files might ignore it,
also this is how it was written in rh_cloud, so maybe we can try to add it:
```rb
s.files = Dir['{app,config,db,lib,locale,webpack}/**/*'] +
            ['LICENSE', 'Rakefile', 'README.md'] +
            ['package.json']
```